conan/cli/command.py

```diff
@@ -100,9 +102,14 @@ class ConanArgumentParser(argparse.ArgumentParser):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
```

Comment:
What if we add the -gc arg in this file so that every command gets it by default? Just like for format

Reply:
It is being added to all commands already, isn't it?

Reply:
If you mean the parsing and handling, we need the ``conan_api``, which doesn't seem easily available here.

Reply:
Oh missread which file was being modified, we actually had the same idea then, 👍